run-tests.sh

```diff
@@ -5,8 +5,21 @@
 
 set -e
 
+PLATFORM="linux/amd64"
```

Comment:
since we have to install libreoffice from source and its available in amd or arm and didnt want to complicate the dockerfile to support both, so force local building to use amd. github uses amd by default. likewise the default lambda architecutre on linux still is amd so all works withotu extra config.